### DISCUSSION TITLE
Build failing with 2.3.0-RC2

### DISCUSSION THREAD

**Discussion by hariohmprasath** [awspring/spring-cloud-aws#100] [Q&A] [answered]
Hi @eddumelendez & @maciejwalkowiak ,
I checked in a minor change to my pull request and I am seeing the build fail as of a sudden. The same build is going through fine in my local laptop.

[https://github.com/awspring/spring-cloud-aws/pull/95](https://github.com/awspring/spring-cloud-aws/pull/95)

Can you please help me with this?

Thanks
Hari

**Comment by arunkpatra** [comment]
Hello @hariohmprasath, Can you please delete artifacts from your `~/.m2/repository` directory on your laptop and try to build once? Theoretically, the exact same behavior that we see at GitHub  CI should be visible in local machines. Once we replicate that, we could take it from there. I'm not sure if the recent release has something to do with it.

**Comment by arunkpatra** [comment]
@hariohmprasath, could you please rebase from https://github.com/awspring/spring-cloud-aws 2.3.x branch once and try to rebuild on your local machine. A release was done by @eddumelendez that updated version to 2.3.0 recently.

**Comment by maciejwalkowiak** [comment, marked as answer]
@hariohmprasath please merge back 2.3.x into your branch. Poms now have SNAPSHOT versions so this sort of not-clean state should not happen again.

**Comment by hariohmprasath** [comment]
Thanks @maciejwalkowiak and @arunkpatra for all the suggestions, rebasing it with 2.3 branch and bumping pom versions to 2.3.1-SNAPSHOT solved the issue. Build is successful now.
